index.bs

```diff
@@ -4043,6 +4063,18 @@ The [=remote end steps=] with |session| and |command parameters| are:
          1. Let |result nodes| be [=locate nodes using accessibility attributes=]
             given |context nodes|, |selector|, and |maximum returned node count|.
 
+      <dt>|type| is the string "<code>container</code>"
```

Comment:
We're not locating by container, we're locating a container by context id. So I think this would make sense if we had the type here as "context" or "containedContext" or something.